### ISSUE TITLE
ZoomIn shortcut (ctrl,+) is not work in my laptop

### ISSUE THREAD

**Issue by kitepad** [stefankueng/BowPad#261]
My laptop is thinkpad nano and os is win11. 

ZoomOut shortcut (ctrl,-) is working. 
And I have a test using notepad3.exe, (ctrl,+) shortcut is working.

**Comment by wmjordan** [comment]
It does not work on mine (win10) either.

**Comment by wmjordan** [comment]
You have to change following line in the _shortcuts.ini_ file:

````ini
cmdZoomIn=Ctrl,+
````
to the following:
````ini
cmdZoomIn=Ctrl,=
````
And don't forget to restart BowPad.

**Comment by kitepad** [comment]
Yes. This work round is work for me. I want to know why. In notepad3's rc file, shortcut is defined by VK_OEM_PLUS and VK_OEM_MINUS for zoom in/out, and it is work.  But I follow this to try in Bowpad, it is not work.

**Comment by wmjordan** [comment]
I am not good at Win32 programming. Let's wait for Stefan.

**Comment by stefankueng** [comment]
you can check the actual codes here: https://docs.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes
so VK_OEM_PLUS would be 0xBB and VK_OEM_MINUS would be 0xBD:

cmdZoomin=Ctrl,0xBB
cmdZoomout=Ctrl,0xBD
